libs/api/domains/cms/src/lib/search/importers/aboutPage.service.ts

```diff
@@ -32,10 +37,14 @@ export class AboutPageSyncService implements CmsSyncProvider<IPage> {
       .map<MappedData | boolean>((entry) => {
         try {
           const mapped = mapAboutPage(entry)
+          const content = extractStringsFromObject({ ...mapped.slices }) // this function only accepts plain js objects
           return {
             _id: mapped.id,
             title: mapped.title,
-            content: extractStringsFromObject({ ...mapped.slices }), // this function only accepts plain js objects
+            content,
+            contentWordCount: content.split(' ').length,
```

Comment:
Content might include multiple spaces, which would throw the count off, we could use regex to mitigate that `/\s+/`